index.html

```diff
@@ -11,39 +11,54 @@
     
 <header>
     <!-- <div class="logo">   -->
-      <div class="nav-items logo"><a href="" id="logo">Sunga</a>
+      <div class="nav-items logo"><a href="#">Sunga</a>
     </div>
     <nav>
         <div class="nav-items menuicon">
           <!-- <div class="menuicon"> -->
         <img class="hamburger" src="./images/Icon - Menu.png" alt="Hamburger menu icon">
         </div>
+
+        <ul class="desktop desk-nav-list">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
     </nav>
+
+    <div class="desktop desk-email-box">
+      <img src="./images/Union.png" alt="Email" class="email-icon">
+    </div>
 </header>
 
 <section class="intro">
     <div class="headline">
-        <h1 class="hello-me">Hey There. I'm Sunga</h1>
-        <h3 class="SD">I'm a Software Developer</h3>
-        <p class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-        <ul class="icons">
-            <li><a href="https://www.github.com/sunga12"><img src="/images/github.svg" class="contact-icon"></a></li>
-            <li><a href="https://www.linkedin.com/in/sungabanja-thawethe-b3419b142"><img src="/images/linkedin.svg" class="contact-icon"></a></li>
-            <li><a href="#"><img src="/images/angelist.svg" class="contact-icon"></a></li>
-            <li><a href="https://twitter.com/OfficialAseT"><img src="/images/twitter.svg" class="contact-icon"></a></li>
-            <li><a href="#"><img src="/images/medium.svg" class="contact-icon"></a></li>
-        </ul>
+      <h1 class="hello-me">Hey There. <br> I'm Sunga.</h1>
+      <h3 class="SD">I'm a Software Developer</h3>
+      <p class="info">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
     </div>
+    <ul class="icons">
+      <li><a href="https://www.github.com/sunga12"><img src="/images/github.svg" class="contact-icon"></a></li>
+      <li><a href="https://www.linkedin.com/in/sungabanja-thawethe-b3419b142"><img src="/images/linkedin.svg" class="contact-icon"></a></li>
+      <li><a href="#"><img src="/images/angelist.svg" class="contact-icon"></a></li>
+      <li><a href="https://twitter.com/OfficialAseT"><img src="/images/twitter.svg" class="contact-icon"></a></li>
+      <li><a href="#"><img src="/images/medium.svg" class="contact-icon"></a></li>
+    </ul>
 </section>
 
 <section class="Recent-works">
 
+  <div class="recent-works-heading">
     <h1 class="recent-works-title">My Recent Works</h1>
-    <hr/>
+  
+    <div class="line"></div>
```

Comment:
- [ ] Kindly Stick to the Figma design as I mentioned above your design is not like the Figma design kindly fix it :+1: 

**Your design**
![Screenshot from 2023-03-30 18-02-08](https://user-images.githubusercontent.com/45226848/228851342-848d09d2-1b66-4452-a324-3a58f1ee24b0.png)

**Figma design**
![Screenshot from 2023-03-30 18-02-55](https://user-images.githubusercontent.com/45226848/228851378-81944ebb-101c-4187-a79e-027f5d7d7455.png)